WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/WaterOrdersAlertNotification.java

```diff
@@ -70,45 +66,42 @@ public String getMessageBodyHTML(Container c, User u)
         final Date now = new Date();
 
         msg.append("This email contains any water orders not marked as completed.  It was run on: " + _dateFormat.format(now) + " at " + _timeFormat.format(now) + ".<p>");
-        findWaterOrdersNotCompleted(c,u,msg,new Date(), false);
-        findWaterOrdersNotCompleted(c,u,msg,new Date(), true);
+        findWaterOrdersNotCompleted(c,u,msg,LocalDateTime.now(), false);
+        findWaterOrdersNotCompleted(c,u,msg,LocalDateTime.now(), true);
 
         return msg.toString();
 
     }
 
-    private void findWaterOrdersNotCompleted(Container c,User u, StringBuilder msg, final Date maxDate, boolean includeFuture){
-        Calendar currentTime = Calendar.getInstance();
-        currentTime.setTime(maxDate);
+    private void findWaterOrdersNotCompleted(Container c,User u, StringBuilder msg, final LocalDateTime maxDate, boolean includeFuture){
+        LocalDateTime currentTime = LocalDateTime.of(maxDate.toLocalDate(), maxDate.toLocalTime());
```

Comment:
If you're just creating a "copy" of the maxDate variable you can just do something like the following since LocalDateTime is immutable

LocalDateTime currentTime = maxDate;

See link for example: https://stackoverflow.com/questions/51842569/best-way-to-clone-java-time-localdatetime

Reply:
modify based on recomendation